config/core/MezaCoreExtensions.yml

```diff
@@ -49,7 +49,9 @@ list:
 
   - name: SemanticCompoundQueries
     composer: "mediawiki/semantic-compound-queries"
-    version: "1.2.0"
+    version: "2.1.0"
+    config: |
+      wfLoadExtension( 'SemanticCompoundQueries' );
```

Comment:
I don't believe this wfLoadExtension.... call under config is necessary. I believe Meza calls wfLoadExtension when it reads this yaml file. I think "config" field can be left blank

Reply:
I can't confirm if it's necessary or not but when I submitted d210b7a, Wiki never loaded the extension until I added the wfLoadExtension in ccee3e5.

I'll try to test this out more and will reply here with the results.